Makefile

```diff
@@ -141,19 +141,19 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 
 deploy: gen-manifests ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
-	kubectl wait --for=condition=Ready pods --all -n cert-manager
+	kubectl wait --for=condition=Ready pods --all -n cert-manager --timeout=60s
```

Comment:
After adding kind target, I received this error:
`make: *** [Makefile:144: deploy] Terminated
error: no matching resources found`
Adding the timeout here instead of the timeout in kind.yaml [here](https://github.com/project-flotta/flotta-operator/pull/106/files#diff-e9c24daafcd1e27b49ed8503aee69248c36d7fc1a47ecbafffe36ba946c61f27L29 fixed the problem. @masayag WDYT?